### PR TITLE
Delete a user token

### PR DESCRIPTION
**Pull request by SarahBA**
closing https://github.com/EUDAT-B2SHARE/b2share/issues/1363

### CODE REVIEW

File: webui/src/components/user.jsx [hunk 1]
```diff
@@ -156,46 +157,47 @@ export const UserProfile = React.createClass({
...
+            serverCache.newUserToken(tokenName, token => {
+                notifications.success("A new Token is created");
+                this.setState({token});
             });
         }
     },
 
     renderToken(t) {
-        return <li className="list-group-item" key={t.id}>{t.name}</li>;
+        return <div key={t.id}> <Token token_id={t.id} token_name={t.name} removeToken={this.removeToken} /> </div>
```
Comment (emanueldima) [inline webui/src/components/user.jsx]: I think we can we remove the topmost div.

File: webui/src/components/user.jsx [hunk 2]
```diff
@@ -156,46 +157,47 @@ export const UserProfile = React.createClass({
...
+        if (!tokenName) {
             notifications.danger('Please choose a name for the new token');
         } else {
             notifications.clearAll();
-            serverCache.newUserToken(this.state.name, token => {
-                const tokenList = this.state.tokenList;
-                tokenList.push(token);
-                this.setState({token, tokenList})
+            serverCache.newUserToken(tokenName, token => {
+                notifications.success("A new Token is created");
```
Comment (SarahBA) [inline webui/src/components/user.jsx]: Right, it's not necessary.

File: webui/src/components/user.jsx (hunk 1 above)
Comment (SarahBA) [inline webui/src/components/user.jsx]: true, updating the PR right now

Comment (SarahBA) [inline webui/src/components/user.jsx]: No wait, although they get a key in 'Token' render method, `key={this.props.token_id}`, each of the elements in the iterator still needs a key in `TokenList`component. That's why I added the div there.

Comment (emanueldima) [inline webui/src/components/user.jsx]: Ok.

**Pull request merged**